node/src/components/consensus/protocols/highway.rs

```diff
@@ -650,7 +653,7 @@ impl<I: NodeIdT, C: Context + 'static> HighwayProtocol<I, C> {
                 }
                 Some(latest_obs) => match latest_obs {
                     Observation::None | Observation::Faulty => {
-                        let index_panorama = format!("correct({:?})", our_seq);
+                        let index_panorama = format!("Count({:?})", our_count.saturating_sub(1));
                         error!(
                             ?vid,
                             ?index_panorama,
```

Comment:
```suggestion
                        error!(
                            ?vid,
                            count=%our_count.saturating_sub(1),
```

Reply:
Also, I'm curious why we are deleting one here?

Reply:
Sorry, subtracting was a mistake! I'll just print the count…